### PR TITLE
Add issue and pull request templates

### PR DESCRIPTION
**Pull request by stormqueen1990**
### Changes
* Add the issue and pull request templates that currently exist in the rancher/rancher repository to this repository.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<!--
+* Custom = Running a docker command on a node
+* Imported = Running kubectl apply onto an existing k8s cluster
+* Hosted = EKS, GKE, AKS, etc
+ * Infrastructure Provider = Rancher provisioning the nodes using different node drivers (e.g. AWS, Digital Ocean, etc)
+-->
+
+### User Information
+- What is the role of the user logged in? (Admin/Cluster Owner/Cluster Member/Project Owner/Project Member/Custom)
+  - If custom, define the set of permissions:
```
Comment (HarrisonWAffel) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: It might be useful to know what version of the provider they're using

**Review by HarrisonWAffel — COMMENTED** [review]
One small suggestion, thanks for raising this PR the templates will be helpful!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (stormqueen1990) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: Good point, I totally overlooked it 😅 
Will update.

Comment (stormqueen1990) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: @HarrisonWAffel I added it under the `### Provider Information` header along with Terraform version, could you please take another look? 🙏

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+## Issue: <!-- link the issue or issues this PR resolves here -->
+<!-- If your PR depends on changes from another pr link them here and describe why they are needed on your solution section. -->
```
Comment (a-blender) [inline .github/pull_request_template.md]: Nit: `...from another PR link them here and describe why in your solution`

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ 
+## Problem
+<!-- Describe the root cause of the issue you are resolving. This may include what behavior is observed and why it is not desirable. If this is a new feature describe why we need this feature and how it will be used. -->
+ 
+## Solution
+<!-- Describe what you changed to fix the issue. Relate your changes back to the original issue / feature and explain why this addresses the issue. -->
```
Comment (a-blender) [inline .github/pull_request_template.md]: Nit: `how` this addresses the issue

**Review by a-blender — CHANGES_REQUESTED** [review]

**Review by HarrisonWAffel — APPROVED** [review]

File: .github/pull_request_template.md (hunk 2 above)
Comment (stormqueen1990) [reply .github/pull_request_template.md]: Updated!

File: .github/pull_request_template.md (hunk 3 above)
Comment (stormqueen1990) [reply .github/pull_request_template.md]: Updated!

**Review by a-blender — APPROVED** [review]

**Pull request merged**